dateparser/date.py

```diff
@@ -326,7 +326,7 @@ class DateDataParser(object):
     locale_loader = None
 
     @apply_settings
-    def __init__(self, languages=None, locales=None, region=None, try_previous_locales=True,
+    def __init__(self, languages=None, locales=None, region=None, try_previous_locales=False, #jayongg updated
```

Comment:
Why did you add this now? I’m a bit confused. At first sight it looks to me like an unrelated, backward-incompatible change.

Reply:
So Sorry!!! I was experimenting and thought I was in a branch.  Let me revert it now